File: Segundo Corte/Python a Java/Circle/Circle.java
public class Circle {
    private double pi = 3.14;
    private double radius;
    private double diameter;
    
    public Circle(double diameter){
        System.out.println("Creating circle with diameter: " + diameter);
        this.radius = diameter/2;
        this.diameter = diameter;
    }
    
    public double circunference(){
        return 2*this.pi*this.radius;
    }
    
    
}

File: Segundo Corte/Python a Java/Circle/Python/Circle.py
class Circle:
    pi = 3.1416
    def __init__(self, diameter):
        print("Creating circle with diameter {d}".format(d=diameter))
        
        self.radius = diameter/2
        
    def circumference(self):
        return self.pi*2*self.radius
medium_pizza = Circle(12)
teaching_table = Circle(36)
round_room = Circle(11460)
print(medium_pizza.circumference())
print(teaching_table.circumference())
print(round_room.circumference())
        